website/docs/guides/docs/docs-markdown-features.mdx

```diff
@@ -26,3 +26,32 @@ I am referencing a [document](doc1.md). Reference to another [document in a fold
 One benefit of this approach is that the links to external files will still work if you are viewing the file on GitHub.
 
 Another benefit, for versioned docs, is that one versioned doc will link to another doc of the exact same version.
+
+## Referencing blog posts
```

Comment:
Hi,

This section looks to me not really useful, as we don't particularly have any "docs to blog" special referencing code.

I'd rather see it renamed "referencing anything else", because a blog post is no different than any other doc/blog/page.

Also, I'd prefer to avoid documenting usage of paths such as `../../blog/2021/01/02/hello-there`, as they are less maintainable.

The doc should rather just say "if you have a page at `/x/y/z`, you can reference it using `[xyz](/x/y/z)`. 

Using absolute paths will likely be more maintainable as it allows you to change the slug of the current doc and move it around without risking to break relative links.